### DISCUSSION TITLE
不能绑定对象吗？标签显示有问题

### DISCUSSION THREAD

**Discussion by Misacky** [irihitech/Ursa.Avalonia#451] [Q&A]
![image](https://github.com/user-attachments/assets/de3c8aac-4e4d-4c45-8107-b4034444bb86)
![image](https://github.com/user-attachments/assets/93628988-e88b-40c8-9e81-613fc14bbfdc)

**Comment by rabbitism** [comment]
设置SelectedItemTemplate

**Comment by Misacky** [comment]
谢谢，后面设置了这个显示出来了，但是binding 了selectitems以后，选中的不会添加到标签中，设置的默认值也无法清除，这个是要咋配置？



---- 回复的原邮件 ----
| 发件人 | Dong ***@***.***> |
| 日期 | 2024年10月30日 11:26 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [irihitech/Ursa.Avalonia] 不能绑定对象吗？标签显示有问题 (Issue #449) |

设置SelectedItemTemplate

**Comment by rabbitism** [comment]
> 谢谢，后面设置了这个显示出来了，但是binding 了selectitems以后，选中的不会添加到标签中，设置的默认值也无法清除，这个是要咋配置？
> 
你的selecteditems需要是observable collection
